src/WCSimConstructMaterials.cc

```diff
@@ -1085,6 +1085,13 @@ void WCSimDetectorConstruction::ConstructMaterials()
    G4double EFFICIENCY_glasscath[NUM] =
      { 0.0, 0.0 };
 
+   // Coated surface properties for new photocathode physics
```

Comment:
Add macro commands to allow variation/fitting studies later?

Reply:
The refractive indices are in general a function of photon energy/wavelength. Need to think of a way to allow variation with a single or few parameters, e.g. using some pre-defined sets and parameteric forms

Reply:
i guess related to Xiaoyue's request below, then are these parameters just assumed constant for all wavelength currently?

do you want to try to implement wavelength dependence and variations in this PR?

Reply:
Just implemented some wavelength dependent sets of refractive indices